lib/bigcommerce/connection.rb

```diff
@@ -39,19 +43,19 @@ def verify_peer=(verify)
     end
 
     def ssl_ca_file=(path)
-      @configuration.ssl_ca_file = path
+      @configuration[:ssl_ca_file] = path
     end
 
     def ssl_client_key=(path,passphrase=nil)
```

Comment:
@pedelman I have updated the code and README to include the new key `ssl_client`. Let me know what you think.